### PR TITLE
thisyahlen/added link for terms and conditions

### PR DESCRIPTION
**Pull request by thisyahlen-deriv**

### CODE REVIEW

File: src/javascript/app/pages/trade/purchase.js [hunk 1]
```diff
@@ -105,6 +105,11 @@ const Purchase = (() => {
                     BinarySocket.wait('get_account_status').then(response => {
```
Comment (mahdiyeh-fs) [inline src/javascript/app/pages/trade/purchase.js:105]: why do we need to get the `get_account_status` response to show the message?

Comment (thisyahlen-deriv) [reply src/javascript/app/pages/trade/purchase.js:105]: If i put it outside, the message doesnt get displayed.

File: src/javascript/app/pages/trade/purchase.js [hunk 2]
```diff
@@ -105,6 +105,11 @@ const Purchase = (() => {
...
                         confirmation_error.setVisibility(1);
                         let message = error.message;
+                        if (/CompanyWideLimitExceeded/i.test(error.code)) {
+                            const redirect = '<a href="https://www.binary.com/en/terms-and-conditions.html?anchor=trading-limits#legal-binary" target="_blank" rel="noopener noreferrer">';
+                            const redirect_close = '</a>';
+                            message = localize('No further trading is allowed on this contract type for the current trading session. For more info, refer to our [_1]terms and conditions[_2].',[redirect, redirect_close]);
+                        }
                         if (/NoMFProfessionalClient/.test(error.code)) {
```
Comment (balakrishna-binary) [inline src/javascript/app/pages/trade/purchase.js:113]:
```suggestion
                        else if (/NoMFProfessionalClient/.test(error.code)) {
```
Error code can match only one code. So, it's better to add `else if`.

Comment (thisyahlen-deriv) [reply src/javascript/app/pages/trade/purchase.js:113]: alrightss

**Review by balakrishna-binary — DISMISSED** [review]

**Review by matin-binary — APPROVED** [review]

File: src/javascript/app/pages/trade/purchase.js (hunk 1 above)
Comment (mahdiyeh-fs) [reply src/javascript/app/pages/trade/purchase.js:105]: you can add this in line 104
```suggestion
else if (/InsufficientBalance/.test(error.code)) {
                    confirmation_error.setVisibility(1);
                    const redirect = '<a href="https://www.binary.com/en/terms-and-conditions.html?anchor=trading-limits#legal-binary" target="_blank" rel="noopener noreferrer">';
                    const redirect_close = '</a>';
                    let message = localize('No further trading is allowed on this contract type for the current trading session. For more info, refer to our [_1]terms and conditions[_2].',[redirect, redirect_close]);
                    CommonFunctions.elementInnerHtml(confirmation_error, message);
                }```

Comment (mahdiyeh-fs) [reply src/javascript/app/pages/trade/purchase.js:105]:
```suggestion
                   else if (/CompanyWideLimitExceeded/.test(error.code)) {
                    confirmation_error.setVisibility(1);
                    const redirect = '<a href="https://www.binary.com/en/terms-and-conditions.html?anchor=trading-limits#legal-binary" target="_blank" rel="noopener noreferrer">';
                    const redirect_close = '</a>';
                    let message = localize('No further trading is allowed on this contract type for the current trading session. For more info, refer to our [_1]terms and conditions[_2].',[redirect, redirect_close]);
                    CommonFunctions.elementInnerHtml(confirmation_error, message);
```

**Review by mahdiyeh-fs — APPROVED** [review]

**Pull request merged**